com.civicrm.hrjobroles/CRM/Hrjobroles/BAO/HrJobRoles.php

```diff
@@ -68,18 +68,24 @@ public static function contactExists($searchValue, $searchField) {
    * @return array
    */
   public static function getDepartmentsList($contractID)  {
+    $result = array();
+
     $queryParam = array(1 => array($contractID, 'Integer'));
     $query = "SELECT cov.id, cov.label
-            FROM civicrm_hrjobroles hrjr
-            LEFT JOIN civicrm_option_value cov
```

Comment:
@AkA84 to get the label of the department from the option value since it was stored in civicrm_hrjobroles table by it's ID .